src/components/Form.js

```diff
@@ -0,0 +1,25 @@
+import React from 'react';
+import {View} from 'react-native';
+import {accessibilityRoleForm, setBrowserAttributes} from '../libs/TextInputUtils';
+
+class Form extends React.Component {
+    componentDidMount() {
+        setBrowserAttributes(this.form, 'method', 'post');
+        setBrowserAttributes(this.form, 'action', '/');
```

Comment:
Is this still required? If so can you add a comment why?

Reply:
I don't know thee reason but it is mentioned on each Password Manager to add these attributes and this is Web standard.